package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "SlickGrid",
-  "version": "3.9.1",
+  "version": "3.9.2",
```

Comment:
could version bumps be done in a separate commit?  makes it easier to see in the history